src/comments.js

```diff
@@ -1,7 +1,24 @@
 "use strict";
 
 var assert = require("assert");
-var types = require("ast-types");
+var types = require("ast-types/fork")([
```

Comment:
I'm gonna pull these out int there own file. Would be nice if that could work without requiring all default ones.